learn/on-premise/aws/install-prerequisites.md

```diff
@@ -69,10 +90,37 @@ sudo apt-get install python3 -y
 
 ## RHEL
 
-Install below softwares on Platform Instance and StudioWorkspace Instance / AppDeployment Instance
+### The ssh user has privileges(root/sudo) or user don't have privileges for install/upgrade utility softwares
+
+#### Platform Instance
+
+- If given ssh user has privileges(root/sudo) or user don't have privileges to install/upgrade. WME Installer will automatically install the Docker software. Internet is not required for Installation in this case.
```

Comment:
wrong sentence. please rewrite this

SSH user performing the install or upgrade needs root/sudo privileges.